app/uk/gov/hmrc/eoricommoncomponent/frontend/forms/models/registration/ContactDetailsModel.scala

```diff
@@ -56,6 +57,24 @@ case class ContactDetailsModel(
     countryCode
   )
 
+  def toRowContactInformation(): ContactInformation = ContactInformation(
```

Comment:
Please do not send contact address even if it is available from REG01

Reply:
Changed